app/src/gui/components/map/center-control-icon.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Creates an SVG icon for the center control button.
+ *
+ * @returns {SVGSVGElement} - The SVG element representing the center control icon.
+ */
+export const CenterControlIcon = () => {
+  const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+  svg.setAttribute('width', '24');
+  svg.setAttribute('height', '24');
+  svg.setAttribute('viewBox', '0 0 24 24');
+
+  const path = document.createElementNS('http://www.w3.org/2000/svg', 'path');
+  path.setAttribute(
+    'd',
+    'M12 8c-2.21 0-4 1.79-4 4s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 6c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2zm8.94-3c-.46-4.17-3.77-7.48-7.94-7.94V1h-2v2.06C6.83 3.52 3.52 6.83 3.06 11H1v2h2.06c.46 4.17 3.77 7.48 7.94 7.94V23h2v-2.06c4.17-.46 7.48-3.77 7.94-7.94H23v-2h-2.06zM12 19c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7z'
+  );
+  path.setAttribute('fill', '#ffffff');
+
+  svg.appendChild(path);
+
+  return svg;
+};
```

Comment:
Is this a maintainable way to create the icon? Can we load it from an svg file instead so it's a bit more visible what the icon is at a glance and IDEs can preview it etc.

Reply:
actioned - I have moved the svg into a separate file